gradle/dependencies.gradle

```diff
@@ -33,9 +33,9 @@ configurations.all {
         force "org.apache.httpcomponents:httpclient:4.5.12"
 
         dependencySubstitution {
-            substitute module("org.mortbay.jetty:jetty") with module("org.eclipse.jetty:jetty-server:9.4.30.v20200611")
-            substitute module("org.mortbay.jetty:jetty-util") with module("org.eclipse.jetty:jetty-util:9.4.30.v20200611")
-            substitute module("org.mortbay.jetty:jetty-http") with module("org.eclipse.jetty:jetty-http:9.4.30.v20200611")
+            substitute module("org.mortbay.jetty:jetty") with module("org.eclipse.jetty:jetty-server:9.4.33.v20201020")
+            substitute module("org.mortbay.jetty:jetty-util") with module("org.eclipse.jetty:jetty-util:9.4.33.v20201020")
+            substitute module("org.mortbay.jetty:jetty-http") with module("org.eclipse.jetty:jetty-http:9.4.33.v20201020")
```

Comment:
Is this a nightly build?

Reply:
I don't believe so, this is just hose eclipse does their versioning.